python/paddle/fluid/tests/unittests/test_gradient_clip.py

```diff
@@ -19,6 +19,25 @@
 import paddle
 import paddle.fluid.core as core
 import paddle.fluid as fluid
+import six
+
+
+def fake_reader_creator(word_dict_size,
```

Comment:
Now that `test_gradient_clip.py` and `test_eager_deletion_dynamic_rnn_base.py` use `fake_reader_creator` as the reader, `fake_reader_creator ` should be place to a separate file.

Reply:
Done.